policy/release/slsa_available.rego

```diff
@@ -0,0 +1,37 @@
+#
+# METADATA
+# title: SLSA - Available
+# description: |-
+#   The SLSA Available requirement states the following:
+#
+#   "The provenance is available to the consumer in a format that the consumer accepts. The
+#   format SHOULD be in-toto SLSA Provenance, but another format MAY be used if both producer
+#   and consumer agree and it meets all the other requirements."
+#
+#   This package only accepts the in-toto SLSA Provenance format.
+#
+package policy.release.slsa_available
```

Comment:
Unclear if this package name (and/or filename) is the right format. Ideas are welcome :)

Reply:
I think this works, the only alternative I could think of would be `policy.release.slsa.available` though I'm not sure if it makes any diffrence